src/react-native/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -2,47 +2,92 @@
 @parent learn-react-native 2
 @outline 3
 
-@description TODO
+@description Learn how to set up your environment for React Native development.
 
 @body
 
 ## Overview
 
-In this section, you will:
+In this section, we will:
 
-- TODO
+- Learn about Node.js
+- Recommend tools to use in React Native development
+- Set up required tools to use in React Native development
 
-## Objective 1: TODO
+## Objective 1: Install prerequisites
 
-TODO
+### Install a code editor
 
-### Concept TODO
+There are a variety of code editors that support React, the most popular is Microsoft’s [Visual Studio Code](https://code.visualstudio.com/). VS Code is available for most operating systems and has extensive support for React and JSX including: code completion, code highlighting, and linting. It’s also used by cloud environments like CodeSandbox and StackBlitz, making it easy to switch among different runtime environments.
 
-TODO
+These VS Code extensions will help you format your code consistently:
+
+- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+### Install Node.js and npm
+
+Our solution requires the [Node.js](https://nodejs.org/) JavaScript runtime environment. Node.js and its included package manager `npm` will be used to do a variety of tasks including: installing required packages, running the development server, executing tests, and building the application for deployment.
+
+### Install React Native and Dependencies
+
+There are several dependencies used to create, build, and deploy a React Native application, and the specifics vary based on your development OS (MacOS, Windows, or Linux) and target OS (Android or iOS). Building for iOS require a MacOS computer, so this training only focuses on building for Android, though most of the techniques apply to all targets. Expo Go is a very convenient way to build simple React Native apps, though we will be using the official React Native CLI for this course, due to Expo Go's limitations.
 
 ### Setup 1
 
-TODO
+This course requires Node.js version 20 (we always suggest using the long-term support releases).
+
+The simplest approach is to download the [Node.js installer](https://nodejs.org/en/download) for your operating system. Any installation of Node.js will also include npm.
+
+Alternatively, you can [install Node.js via package manager](https://nodejs.org/en/download/package-manager); this is the reccomendation for most professional setups.
```

Comment:
I want this section to be as extensive as what we have in the [RN PMO README](https://github.com/bitovi/react-native-pmo?tab=readme-ov-file#setup), so I’ll come back and fix typos like `reccomendation`.

Reply:
https://github.com/bitovi/academy/blob/main/src/react-vite/02-setting-up-your-environment/setting-up-your-environment.md#verify-1